### PR TITLE
fix overlay isVisible checkbox remains checked

### PR DESCRIPTION
**Pull request by ankushdutt**
fixes: #61 

Edit: Finally found a way to hook props to the children using react-view AST. Now it works for both on clicking backdrop and TouchableOpacity

### CODE REVIEW

File: src/content/Overlay/overlay.playground.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const OverlayPlayground = () => {
     props: {
       children: {
         type: PropTypes.ReactNode,
-        value: `<Text>Some content</Text><TouchableOpacity onPress={()=>setIsVisible(!isVisible)}><Text>Click to close</Text></TouchableOpacity>`,
```
Comment (tewarig) [inline src/content/Overlay/overlay.playground.jsx:16]: removing touchable opacity will cause a problem when fullscreen is checked. there would be no option to go out.

**Review by tewarig — COMMENTED** [review]
this pr deployment has this problem. if we go in fullscreen mode there would be a problem in going out. 
newPr
just add back touchableOpacity.. so there is a way to close this.
oldPr

**Comment by ankushdutt** [comment]
Gotcha @tewarig, thanks! Added that back

**Comment by pranshuchittora** [comment]
This works, great.
But can we do this with a simple state hook -> https://react-view.netlify.app/?path=/story/useview--state-hook

**Comment by ankushdutt** [comment]
> But can we do this with a simple state hook -> https://react-view.netlify.app/?path=/story/useview--state-hook

No, we can't. This only works for the top level prop (like `onBackdropPress`). Here, `TochableOpacity` is inside of chidren. So, we must correctly target the `onPress` handler using the react-view AST.

**Review by pranshuchittora — APPROVED** [review]
LGTM :)

**Comment by pranshuchittora** [comment]
Thanks

**Pull request merged**